src/lib/vm-manager-hoc.jsx

```diff
@@ -119,6 +122,8 @@ const vmManagerHOC = function (WrappedComponent) {
         return {
             fontsLoaded: state.scratchGui.fontsLoaded,
             isLoadingWithId: getIsLoadingWithId(loadingState),
+            locale: state.locales.locale,
```

Comment:
locale and messages should also be added where we are omitting props to prevent unnecessary props from being passed down to children (in `render` between the two eslint disable comments)

Reply:
Fixed!